src/main/java/com/onarandombox/MultiverseCore/world/SimpleMVWorldManager.java

```diff
@@ -104,23 +83,27 @@ public SimpleMVWorldManager(
         this.worlds = new ConcurrentHashMap<String, MVWorld>();
     }
 
+
     /**
      * {@inheritDoc}
      */
     @Override
     public void getDefaultWorldGenerators() {
         this.defaultGens = new HashMap<>();
-        File[] files = server.getWorldContainer().listFiles((file, s) -> s.equalsIgnoreCase("bukkit.yml"));
-        if (files != null && files.length == 1) {
-            FileConfiguration bukkitConfig = YamlConfiguration.loadConfiguration(files[0]);
-            if (bukkitConfig.isConfigurationSection("worlds")) {
-                Set<String> keys = bukkitConfig.getConfigurationSection("worlds").getKeys(false);
-                for (String key : keys) {
-                    defaultGens.put(key, bukkitConfig.getString("worlds." + key + ".generator", ""));
-                }
+
+        if (plugin.getBukkitConfig() == null) {
```

Comment:
Is that change relevant to the /mv dumps command? Also I recall you said getBukkitConfig() is only supported on paper or smt?

Reply:
I needed the Bukkit.yml file and server.properties for the dump so I felt it should be abstracted there too. It also makes it much easier to edit when the issue is fixed